### DISCUSSION TITLE
Adding a global hotkey for sending a request.

### DISCUSSION THREAD

**Discussion by khusan-dev** [Kong/insomnia#5786] [Ideas]
Right now I'm using a macro for [AutoHotkey](https://www.autohotkey.com/)
<details>
  <summary>AHK macros</summary>

Sending a request when Ctrl+Q is pressed

```
#NoEnv  ; Recommended for performance and compatibility with future AutoHotkey releases.
; #Warn  ; Enable warnings to assist with detecting common errors.
SendMode Input  ; Recommended for new scripts due to its superior speed and reliability.
SetWorkingDir %A_ScriptDir%  ; Ensures a consistent starting directory.

^q::
If WinExist( "ahk_exe Insomnia.exe" )
{
	ControlFocus, , ahk_exe Insomnia.exe
	ControlSend, , ^{enter}, ahk_exe Insomnia.exe
}
return
```

</details>

It works well. But I wish this feature was already built into Insomnia.

**Comment by khusan-dev** [comment]
This is very handy when you have multiple monitors and one of them has Insomnia set to full screen. Thus, you can not distract yourself from coding and test the API by pressing a hotkey.

**Comment by khusan-dev** [comment]
I want to explain why. It’s much more pleasant not to reach for the mouse and change windows, but to press the hotkey without “looking up from the code” and see the result of the request.

**Comment by khusan-dev** [comment]
#refresh

**Comment by subnetmarco** [comment]
Right now I can send a request by pressing CMD+ENTER. You can configure the shortcuts in the Preferences.

**Reply by khusan-dev** [reply]
I know that there is a hotkey that sends a request, but it works when the Insomnia program window is active. I meant a global hotkey that would work regardless of which window is active.
